### PR TITLE
Added items

### PR DESCRIPTION
**Pull request by Ashlee-Benson**
Added items

### CODE REVIEW

File: EMIP/1-1000/EMIP00307.xml [hunk 1]
```diff
@@ -60,6 +60,14 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                         </extent>
                      </supportDesc>
                   </objectDesc>
+                  <decoDesc>
+                    <decoNote type="miniature" xml:id="d01"><locus target="#ir"/><desc><ref type="authFile" corresp="AT1035HolyMan"/>,<persName ref="PRS4585Georgeo"/> and the <term key="dragon"/> with caption <q>“How he rescued Berutiwit,” “Saint George,” and “dragon.”</q></desc></decoNote>
```
Comment (DenisNosnitsin1970) [inline EMIP/1-1000/EMIP00307.xml]: Could you type in the legends?

File: EMIP/1-1000/EMIP00307.xml [hunk 2]
```diff
@@ -60,6 +60,14 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
...
+                    <decoNote type="miniature" xml:id="d02"><locus target="#ivv(erso)"/><desc><ref type="authFile" corresp="AT1031VirginAndChild"/> attended by two angels</desc></decoNote>
+                    <decoNote type="miniature" xml:id="d03"><locus target="#46r"/><desc><ref type="authFile" corresp="AT1082Angel"/> The Angel <persName ref="PRS4377Gabriel"/> rescues the three young men in the fiery furnace. with caption <q>“How Saint <persName ref="PRS7049Michael"/> rescued the children,” “rescued the children from the fiery furnace.”</q></desc></decoNote>
```
Comment (DenisNosnitsin1970) [inline EMIP/1-1000/EMIP00307.xml]: Please type in the legends in Ethiopic.

Comment (DenisNosnitsin1970) [inline EMIP/1-1000/EMIP00307.xml]: I think better is "three youths".  "...the fiery furnace**,** with caption...

File: EMIP/1-1000/EMIP00309.xml [hunk 3]
```diff
@@ -59,6 +59,18 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                         </extent>
                      </supportDesc>
                   </objectDesc>
+                  <decoDesc>
+                    <decoNote type="miniature" xml:id="d01"><locus target="#ir(ecto)"/><desc> Crude drawing of a face above a ḥaräg</desc></decoNote>
+                    <decoNote type="miniature" xml:id="d02"><locus target="#iv(erso)"/><desc><ref type="authFile" corresp="AT1066DavidHarp"/></desc></decoNote>
+                    <decoNote type="miniature" xml:id="d03"><locus target="#29r"/><desc><ref type="authFile" corresp="AT1031VirginAndChild"/> attended by the angels <persName ref="PRS7049Michael"/> and <persName ref="PRS4377Gabriel"/></desc></decoNote>
+                    <decoNote type="miniature" xml:id="d04"><locus target="#59r"/><desc><ref type="authFile" corresp="AT1035HolyMan"/>,<persName ref="PRS4585Georgeo"/> and the <term key="dragon"/></desc></decoNote>
+                    <decoNote type="miniature" xml:id="d05"><locus target="#78v"/><desc><ref type="authFile" corresp ="AT1030Trinity"/> surrounded by the Four Living Creatures. with caption <q>“Holy trinity with four creatures.”</q></desc></decoNote>
```
Comment (DenisNosnitsin1970) [inline EMIP/1-1000/EMIP00309.xml]: "Living Creatures**,** with"

File: EMIP/1-1000/EMIP00314.xml [hunk 4]
```diff
@@ -60,6 +60,10 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                         </extent>
                      </supportDesc>
                   </objectDesc>
+                  <decoDesc>
+                  <decoNote type="miniature" xml:id="d01"><locus target="#ii v(erso)"/><desc> Drawing, in pencil, of an angel</desc></decoNote>
+                  <decoNote type="miniature" xml:id="d02"><locus target="#178v"/><desc> Crude drawing, in pencil, of an equestrian saint. with caption <q>“Angel.”</q> Also, pen trials.</desc></decoNote>
```
Comment (DenisNosnitsin1970) [inline EMIP/1-1000/EMIP00314.xml]: Comma before "with", here and in some other descriptions.

File: EMIP/1-1000/EMIP00317.xml [hunk 5]
```diff
@@ -39,23 +40,23 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                   <summary>School Chants, የቃል፡ ትምህርት፡</summary>
                     <msItem xml:id="ms_i1"><locus from="1r" to="11v"/>
                     <textLang mainLang="gez">Gə‘əz</textLang>
+                    <title>Chants called Mästägabǝ’<foreign xml:lang="gez">መስተጋብዕ፡</foreign></title>
                     <incipit xml:lang="gez">ዕዝል: መስተጋብዕ። ዘሰኑይ: ወለቡ: ጽራኅየ: ወአጽምዓኒ: ቃለ ስእለትየ: ከመ: ወልታ: ሥሙር: ከለልከነ:</incipit>
-                    <note>Chants called Mästägabǝ’</note>
                   </msItem>
                   <msItem xml:id="ms_i2"><locus from="11v" to="17v"/>
+                  <title>Chants called Qalǝyä aṣmǝ`<foreign xml:lang="gez">ቃልየ፡ አጽምዕ፡</foreign></title>
```
Comment (DenisNosnitsin1970) [inline EMIP/1-1000/EMIP00317.xml]: Please use the BM transliteration, https://betamasaheft.eu/Guidelines/?q=transliteration&id=transliteration-principles

File: EMIP/1-1000/EMIP00317.xml [hunk 6]
```diff
@@ -39,23 +40,23 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
...
                   <textLang mainLang="gez">Gə‘əz</textLang>
                   <incipit xml:lang="gez">ቃልየ: አጽምእ: እግዚኦ: ሃሌ: ሉያ: ሃሌ: ሉያ: ወለቡ: ጸራኅየ: (sic) ሃሌ: ሉያ: ሃሌ: ሉያ: ወአጽምዓኒ: ንጉሥየኒ: ወአምላኪየኒ:..</incipit>
-                  <note>Chants called Qalǝyä aṣmǝ`</note>
```
Comment (DenisNosnitsin1970) [inline EMIP/1-1000/EMIP00317.xml]: Please use the BM system, here and below, https://betamasaheft.eu/Guidelines/?q=transliteration&id=transliteration-principles

File: EMIP/1-1000/EMIP00318.xml [hunk 7]
```diff
@@ -52,9 +52,10 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
             <msItem xml:id="ms_i1.3"><locus from="24v" to="54r"/><title type="complete" ref="LIT1097Ammest#Resurrection"/>
             <textLang mainLang="gez">Gə‘əz</textLang>
           </msItem>	<msItem xml:id="ms_i1.4"><locus from="54r" to="58r"/>
+          <title>On the Faith of the Orthodox Church</title>
           <textLang mainLang="gez">Gə‘əz</textLang>
           <incipit xml:lang="gez">ነቢያት: በትንቢት: ሐዋርያት: በስብከት: ያስተማሩት: ለዓለም: ሁሉ: ይህንን: ተዋህዶ: ሃይማኖት: ነው። ነቢያት: እንዴት: እያሉ: አስተማሩ: ቢሉ: ኢሳይያስ: እንደተናገረው: ናሁ: ድንግል: ትፀንስ: ወትወልድ: ወልደ: እያሉ: አስተማሩ:...</incipit>
```
Comment (DenisNosnitsin1970) [inline EMIP/1-1000/EMIP00318.xml]: Three dots at the end of incipit are not needed.

File: EMIP/1-1000/EMIP00317.xml [hunk 8]
```diff
@@ -39,23 +40,23 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
...
                     <msItem xml:id="ms_i1"><locus from="1r" to="11v"/>
                     <textLang mainLang="gez">Gə‘əz</textLang>
+                    <title>Chants called Mästägabǝ’<foreign xml:lang="gez">መስተጋብዕ፡</foreign></title>
                     <incipit xml:lang="gez">ዕዝል: መስተጋብዕ። ዘሰኑይ: ወለቡ: ጽራኅየ: ወአጽምዓኒ: ቃለ ስእለትየ: ከመ: ወልታ: ሥሙር: ከለልከነ:</incipit>
-                    <note>Chants called Mästägabǝ’</note>
                   </msItem>
                   <msItem xml:id="ms_i2"><locus from="11v" to="17v"/>
+                  <title>Chants called Qalǝyä aṣmǝ`<foreign xml:lang="gez">ቃልየ፡ አጽምዕ፡</foreign></title>
                   <textLang mainLang="gez">Gə‘əz</textLang>
                   <incipit xml:lang="gez">ቃልየ: አጽምእ: እግዚኦ: ሃሌ: ሉያ: ሃሌ: ሉያ: ወለቡ: ጸራኅየ: (sic) ሃሌ: ሉያ: ሃሌ: ሉያ: ወአጽምዓኒ: ንጉሥየኒ: ወአምላኪየኒ:..</incipit>
```
Comment (DenisNosnitsin1970) [inline EMIP/1-1000/EMIP00317.xml]: Sic should be like this: <sic></sic>

File: EMIP/1-1000/EMIP00324.xml [hunk 9]
```diff
@@ -45,10 +45,10 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                   <msItem xml:id="ms_i2"><locus from="69v" to="81v"/><title type="complete" ref="LIT2736RepCh20"/>
                   <incipit xml:lang="gez">ሰላም: ለህላዌክሙ: ዘኢይመውዕ: (sic) ህላውያተ:</incipit>
```
Comment (DenisNosnitsin1970) [inline EMIP/1-1000/EMIP00324.xml]: <sic></sic>, here and elsewhere.

File: EMIP/1-1000/EMIP00333.xml [hunk 10]
```diff
@@ -37,7 +38,26 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                </msIdentifier>
                <msContents>
                   <summary>Psalter, ዳዊት፡, Absolution of the Son, ፍትሐት፡ ዘወልድ፡, and Asmat Prayers</summary>
-                  <msItem xml:id="ms_i1">
+                  <msItem xml:id="ms_i1"><locus from="iv(erso)" to="ivv(erso)"/><title type="complete" ref="LIT2260salota"/>
```
Comment (DenisNosnitsin1970) [inline EMIP/1-1000/EMIP00333.xml]: ms_i1 looks like a text for "additions"

File: EMIP/1-1000/EMIP00334.xml [hunk 11]
```diff
@@ -37,9 +38,20 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
...
                <msContents>
                   <summary>Psalter, ዳዊት፡, Table Prayer</summary>
-                  <msItem xml:id="ms_i1">
+                  <msItem xml:id="ms_i1"><locus from="1r" to="180v"/><title type="complete" ref="LIT2701Dawit"/>
                      <textLang mainLang="gez">Gə‘əz</textLang>
                   </msItem>
+                  <msItem xml:id="ms_i2"><locus from="181v" to="183r"/>
+                  <title>Table Prayer, “We Beseech You, the Merciful,”<foreign xml:lang="gez">ሰአልናከ: መሐሪ:</foreign></title>
+                   <textLang mainLang="gez">Gə‘əz</textLang>
+                   <incipit xml:lang="gez">ሰአልናከ: መሐሪ: ሰአልናከ: ፈጣሪ: ሰአልናከ: ገባሪ: ለዘሰዓለከ: ኢትትፈለጥ: ወኢትትሌለይ: የማንከ: ነቅዓ: በረከት: የማንከ: ፈልፈለ: ፍሥሐ: ወሐሤት: ባርከነ: ክርስቶስ: በበረከተ: ሰማያውያን: ወምድራውያን:...</incipit>
```
Comment (DenisNosnitsin1970) [inline EMIP/1-1000/EMIP00334.xml]: Prayer before meal LIT5069Prayer

File: EMIP/1-1000/EMIP00339.xml [hunk 12]
```diff
@@ -37,8 +38,40 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
...
                   <summary>Four Gospels of Empress Zäwditu; </summary>
-                  <msItem xml:id="ms_i1">
+                  <msItem xml:id="ms_i1"><locus from="1r" to="51r"/><title type="complete" ref="LIT1558Matthew"/>
+                  <textLang mainLang="gez">Gə‘əz</textLang>
+                  <incipit xml:lang="gez">ብሥራተ: አብ: ትሩፍ: ማቴዎስ: ሐዋርያ: ፩እም፲ወ፪ቱ: ሐዋርያት: በረከተ: አምላኩ: ውድስት። ወሀብተ: ጸሎቱ: ውክፍት: ተሀሉ: ምስለ: ንግሥትነ: አስካለ: ማርያም: ወምስለ: ፍቁራ: ወራሴ: መንግሥት: ኃይለ: ሥላሴ: ለዓለመ: ዓለም: አሜን።</incipit>
+                </msItem>
+                <msItem xml:id="ms_i1.1"><locus from="51r" to="51r"/>
+                <textLang mainLang="gez">Gə‘əz</textLang>
+                <title>Conclusion of the Gospel of Matthew</title>
+                <incipit xml:lang="gez">መልዓ; ብሥራተ: ማቴዎስ: ሐዋርያ: ወኮነ: ጸሐፎ: በምድረ: ፍልስጥዔም: በአስተሐምሞ: ቅዱስ: እብራይስጣዊ: እምድኅረ: ዕርገቱ: ለእግዚእነ: ኢየሱስ ክርስቶስ: በአጶርግዮ: ለኀበ: ሰማይ: በሰማኒቱ: ዓመታት: በቀዳሚሁ: አም: ዘእምቀላውዴዎስ: ቄሣር: ንጉሠ: ሮም: ወስብሐት: ለእግዚአብሔር: ለዓለመ: ዓለም። አሜን።</incipit>
```
Comment (DenisNosnitsin1970) [inline EMIP/1-1000/EMIP00339.xml]: Translation should be added  through 
                  <foreign xml:lang="en">
                  ...... </foreign>
                  </incipit>
 See https://betamasaheft.eu/Guidelines/?q=translation&id=manuscriptContents

File: EMIP/1001-2000/EMIP01971.xml [hunk 13]
```diff
@@ -57,6 +74,64 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
...
+                  <desc>The word God is not rubricated in the first three works</desc>
+                  <foreign xml:lang="gez">ማርያም፡</foreign>
+                  </seg>
+                  <desc>The word Mary is rubricated in the final two works</desc>
+                  <desc>Columetric layout of text: <locus target="#138rv"/> (tenth Biblical Canticle)</desc>
+                  </handNote>
+                  </handDesc>
+                  <decoDesc>
+                  <decoNote type="frame" xml:id="d01">
+                  Decorative designs:
```
Comment (DenisNosnitsin1970) [inline EMIP/1001-2000/EMIP01971.xml]: These are elaborate ornamental bands, nicely done (a distinctive feature of a special group of Psalters); I would suggest to write them out, specifying a little bit (motifs: floral, zoomorphic, anthropomorphic).

File: EMIP/1001-2000/EMIP01975.xml [hunk 14]
```diff
@@ -57,6 +65,60 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
...
+                    </item>
+                    <item xml:id="e02">
+                    <desc>
+                    Identified in the insurance hand list as Psalter 10
+                    </desc>
+                    </item>
+                    <item xml:id="e03">
+                    <desc>
+                    Navigation system:
+                    purple yarn is sewn into the fore edge of <locus target="#16 #36 #44 #72 #119 #138"/>;
```
Comment (DenisNosnitsin1970) [inline EMIP/1001-2000/EMIP01975.xml]: We describe it as leaf string marker (/leaf tab marker), under binding.  
https://betamasaheft.eu/Guidelines/?q=string&id=bindingDescription

File: EMIP/1001-2000/EMIP01976.xml [hunk 15]
```diff
@@ -57,6 +61,78 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
...
+                  </decoDesc>
+                  <additions>
+                  <list>
+                    <item xml:id="a01">
+                    <desc type="GuestText">
+                    <locus target="#ir(ecto) #iv(erso) #iir(ecto) #iiv(erso) #iiir(ecto) #iiiv(erso) #ivr(ecto) #ivv(erso)"/>": <title type="complete" ref="LIT2285Saqoqa"/>
+                    </desc>
+                    </item>
+                    <item xml:id="a02">
+                      <desc type="GuestText">
```
Comment (DenisNosnitsin1970) [inline EMIP/1001-2000/EMIP01976.xml]: This is LIT2733RepCh17

File: EMIP/1001-2000/EMIP01969.xml [hunk 16]
```diff
@@ -57,6 +65,80 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
...
+                  <desc>The word Mary is rubricated in the final two works. </desc>
+                  </seg>
+                  <desc>Columetric layout of text: <locus target="#110v #111r"/> (Ps. 150); <locus target="#119v #119r #120v #120r"/> (tenth Biblical Canticle)</desc>
+                  </handNote>
+                  </handDesc>
+                  <decoDesc>
+                  <decoNote type="miniature" xml:id="d01"><locus target="#148v"/><desc> of a rider on a horse</desc></decoNote>
+                  <decoNote type="frame" xml:id="d02">
+                  Decorative designs:
+                  <locus target="#1r #13v #20r #40r #48v #72r #105r"/> (haräg)
```
Comment (DenisNosnitsin1970) [inline EMIP/1001-2000/EMIP01969.xml]: I would not use "decorative design" for "harag", I would use "ornamental band" instead (crude, elaborate, monochrome, bicolour, polychrome).

**Comment by PietroLiuzzo** [comment]
@Ashlee-Benson please commit your updates to this branch for it to be merged. thanks

**Comment by PietroLiuzzo** [comment]
It does not seem that the corrections suggested from Denis have been made here, instead I see a mysterious merge from master into this branch. was the agreement not to do them for now?

**Comment by PietroLiuzzo** [comment]
ok, Denis kindly informed me that the latest exchange is the one between me and @Ralph-Lee-UK . it would be extremely useful not to delegate parts of the conversations to emails, if possible. So I will investigate the merge from master and then merge this branch into master if all is ok

**Pull request merged**